docs/beginner/tutorial2-swapchain/README.md

```diff
@@ -145,9 +145,14 @@ If we want to support resizing in our application, we're going to need to recrea
 ```rust
 // impl State
 fn resize(&mut self, new_size: winit::dpi::PhysicalSize<u32>) {
+    // Check if we are positive for the new size value
+    // otherwise when minimise the window may crash
+    let safe_width = if new_size.width > 0 { new_size.width } else { 1 };
```

Comment:
Maybe it would be better to fix on `winit` side with `with_min_inner_size`?

https://docs.rs/winit/0.25.0/winit/window/struct.WindowBuilder.html#method.with_min_inner_size

Reply:
I will try to see if this work and correct the error, if this work i think your version will be more clean to use a native function of the winit crate than using this little trick. 

I'm returning to you after trying, Best regards.

Reply:
> Maybe it would be better to fix on `winit` side with `with_min_inner_size`?
> 
> https://docs.rs/winit/0.25.0/winit/window/struct.WindowBuilder.html#method.with_min_inner_size

So after a test, this function is used if you don't want to resize your window below a certain size but it didn't stop crashing when minimizing it, i think the little trick is what work the most and easy to implement.